lib/ansible/modules/cloud/azure/azure_rm_managed_disk.py

```diff
@@ -85,6 +85,15 @@
     tags:
         description:
             - Tags to assign to the managed disk.
+    zones:
+        description:
+            - "Allowed values: 1, 2, 3, None."
+        choices:
+            - 1
+            - 2
+            - 3
+            - None
```

Comment:
"None" here may be a bit confusing.
Choices are numeric here.... but strings afterwards

Reply:
@zikalino How do you think this should be changed? changed like the following:
```e.g.
            - "Allowed values: 1, 2, 3."
        choices:
            - 1
            - 2
            - 3